assets/resource.h

```diff
@@ -0,0 +1 @@
+#define IDI_APPICON 101
```

Comment:
Its mostly like this because its how I've seen it done, and it may only be one line atm, but you can add a lot more information into a windows executable here.